ansible/files/bootstrap/telemetry.sh

```diff
@@ -152,6 +152,12 @@ bash /mnt/$ANACONDA_SCRIPT -b -p $ANACONDA_PATH
 PIP_REQUIREMENTS_FILE=/tmp/requirements.txt
 aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/python-requirements.txt $PIP_REQUIREMENTS_FILE
 $ANACONDA_PATH/bin/pip install -r $PIP_REQUIREMENTS_FILE
+
+conda create -n zeppelin python=3.6 cairo pillow -y -q
+source activate zeppelin
+pip install python-zeppelin
```

Comment:
The answer (via IRC) was no b/c python-zeppelin requires Python 3.x since it uses cairosvg. One option discussed was to add a python3-requirements.txt file which pyup can also read.